plugin

```diff
@@ -1,5 +1,6 @@
 [
   "9a4gl/lovelace-centrometal-boiler-card",
+  "0Paul89/vienna-transport-card",
```

Comment:
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>